index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Website</title>
+</head>
+<body >
+    <nav class="header-logo">
+        <img src="images/My Logo.svg" alt="" class="logo">
+        <img src="images/Enabled.png" alt="" class="image-icon"></nav>
+         
+    </nav>
+
+    <section class="all-body">
+        <section class="main">
+            <h2 class="title">I'm Mariane Glad to see you!</h2>
+            <p class="intro">I'm a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't 
+                hestiate to contact me.</p>
+        </section>
+
+        <section class="connect">
+
+            <div class="lets-connect">
+                <p>LET'S CONNECT</p>
+            </div>
+
+            <ul>
+            <img src="images/Vector.png" class="icons">
+            <img src="images/Vector (1).png" class="icons"><img src="images/Vector (2).png" class="icons">
+            <img src="images/Vector (3).png" class="icons">
+            <img src="images/Vector (4).png" class="icons">
+            <img src="images/Vector (5).png" class="icons">
+            </ul>
+
+        </section>
```

Comment:
- [ ] As the previous reviewer said, he notice you wrap part of your heading into a `section` tag, Please notice everything under the heading is a single section and should be treated as that. Please wrap all your section in a section tag instead of `div`. 👍 
- [ ] As the previous reviewer said, Please also don't forget to use the right `font family` and `size` for your texts, especially your heading text, as you can see your design is not fit into a single view on a `375px` screen size.
- [ ]  As the previous reviewer said 😃, aside from that, Your background image is not aligned to take the whole width of the display, If you notice on the figma design, you will see the background take all the space from left to right. Please make sure to align your background properly.

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | | ![h](https://user-images.githubusercontent.com/57408419/219827972-85dbe4af-0d88-40df-a3a3-b09ec504514d.png) | | ![he](https://user-images.githubusercontent.com/57408419/219828009-1c75869b-e51d-431f-acbd-78ec97ef8f47.jpg) | |